### ISSUE TITLE
What are the differences between the 5 models?

### ISSUE THREAD

**Issue by tpq** [Illumina/SpliceAI#59]
Hi there,

What are the differences between the 5 models stored in the `~/spliceai/models` directory? They appear to have the same architecture when inspected in Keras. Were they trained using different initializations? Or does something else differentiate them?

Thanks!

**Comment by kishorejaganathan** [comment]
The models have the same architecture, and were the results of five different training processes. The differences in the weights are because of the randomness involved in the training process like initializations and the order in which the data is presented (i.e. shuffling of training data).

**Comment by tpq** [comment]
Thanks!!
